examples/saas-clients/amazons3/working-with-amazons3-client/README.md

```diff
@@ -0,0 +1,422 @@
+# Working with Amazon S3 client
+
+## Overview
+The Ballerina Amazon S3 client provides connectivity to Amazon Simple Storage Service API. It allows users to interact 
+with Amazon S3 API to create, store and download and use data with other services. 
```

Comment:
```suggestion
with the Amazon S3 API to create, store, download, and use data with other services. 
```

Generally it is best to use a serial comma (comma before and) in lists.

Reply:
Fixed.